test/test_nestedtensor.py

```diff
@@ -2423,6 +2444,21 @@ def grad_test_func(a, b, c):
         data = (a, b, c)
         assert gradcheck(grad_test_func, inputs=data, check_batched_grad=False)
 
+    def test_nested_from_values_and_offsets_backward(self, device):
+        batch_size = 8
+        embedding_dim = 2
+        lengths = torch.randint(low=0, high=10, size=(batch_size,), dtype=torch.int32, device=device)
+        offsets = lengths.cumsum(0)
+        offsets = torch.cat([torch.tensor([0], dtype=torch.int32, device=device), offsets])
+        values = torch.randn((offsets[-1], embedding_dim), dtype=torch.float64, device=device, requires_grad=True)
+
+
+        def grad_test_func(values, offsets):
+            nt = torch.nested._from_values_and_offsets(values, offsets)
```

Comment:
Extend this graph a little and do some operations on the nt


As well try a mutating op in place. I think we have relu_

If any fail I think that's still okay but we need to track them and Xfail